README.md

```diff
@@ -391,6 +391,16 @@ auto result = task->get(); // The final task's result remains valid
 The `releaser` also accepts an executor that gives control over _where_ a task's
 result is released.
 
+## Using transwarp with tipi.build
+
+`transwarp` can be easily used in [tipi.build](https://tipi.build) projects simply by adding the following entry to your `.tipi/deps`:
+
+```json
+{
+    "bloomen/transwarp": { }
```

Comment:
Where would transwarp be located or come from when integrated into a tipi project?

Reply:
We pull the sources straight from your github repository. Basically `bloomen/transwarp` is a shortcut for `github.com/bloomen/transwarp`. We currently default to pulling the latest revision (live at HEAD 💯 ) but it's easy to pin a version by adding an `@` section to the dependency configuration:

```json
{
    "bloomen/transwarp": { "@": "release-tag-ex-2.2.2" }
}
```